src/Hintman/Hint/TrailingNewlines.hs

```diff
@@ -0,0 +1,41 @@
+module Hintman.Hint.TrailingNewlines
+       ( getTrailingNewlinesComments
+       ) where
+
+import Hintman.Core.Hint (HintType (TrailingNewlines), Line (..), toLines)
+import Hintman.Core.PrInfo (ModifiedFile (..))
+import Hintman.Core.Review (Comment (..))
+import Hintman.Hint.Position (getTargetCommentPosition)
+
+import qualified Data.Text as T
+
+
+-- | Creates 'Comment's on removing trailing newlines in all modified files.
+getTrailingNewlinesComments :: [ModifiedFile] -> [Comment]
+getTrailingNewlinesComments = foldMap getFileTrailingNewlineComments
+
+getFileTrailingNewlineComments :: ModifiedFile -> [Comment]
+getFileTrailingNewlineComments ModifiedFile{..} = maybeToMonoid $
+    mapMaybe createComment . getTrailingSpaceLines . toLines . decodeUtf8
+    <$> mfContent
+  where
+    -- TODO: remove duplication with 'getTrailingSpacesComments'
+    -- | Create a 'Comment' from all necessary data.
+    createComment :: Line -> Maybe Comment
+    createComment Line{..} = do
+        let commentPath = fileNameText
+        commentPosition <- getTargetCommentPosition mfDelta lineNumber
+        let commentBody = unlines [ "```suggestion", "```" ]
```

Comment:
We should refactor this later to some function as this is the reusable part. Can add this to the tech debt.